k8s/pkg/apis/longhorn/v1beta2/backup.go

```diff
@@ -51,6 +51,9 @@ type BackupSpec struct {
 	// Can be "full" or "incremental"
 	// +optional
 	BackupMode BackupMode `json:"backupMode"`
+	// The backup block size
+	// +optional
+	BackupBlockSize string `json:"backupBlockSize"`
```

Comment:
Why not use `int64`?

Reply:
To align the `BackupStatus.Size`. Both `string` or `int64` are OK to me.

Reply:
Got it. Not sure why `status. Size` is a string, but I prefer the `int64` instead.

Reply:
Thanks. Will change to `int64`.